Content/Items/Magnet/Weapons.GrayGoo.cs

```diff
@@ -0,0 +1,435 @@
+using StarlightRiver.Content.Buffs.Summon;
+using StarlightRiver.Content.Dusts;
+using System.Collections.Generic;
+using System.Linq;
+using Terraria.DataStructures;
+using Terraria.ID;
+using static Terraria.ModLoader.ModContent;
+
+namespace StarlightRiver.Content.Items.Magnet
+{
+	public class GrayGooDustData //Has to be a class so we can pass by reference
```

Comment:
structs can be passed by refference with the ref keyword?

Reply:
structs are evil

Reply:
While a class is probably preferrable we shouldn't leave misleading comments.